src/api/HttpHelper.tsx

```diff
@@ -99,7 +101,7 @@ function getOrGenerateUUid(): string {
 }
 function findForEqualSentRequest(request: ApiRequest) {
     return requests.filter(r => {
-        return r.type === request.type && r.data === request.data
+        return r.type === request.type && r.data === request.data && r.mId !== request.mId;
```

Comment:
what is this check? It will lead to resending of exactly matching `mId`s.

Reply:
The method should find "equal" Requests. Not the exact request it got as a parameter. This parameter is handled before.